src/components/about-us/section-05/index.js

```diff
@@ -0,0 +1,465 @@
+    this.setState({ timelineScrolling: false })
+  }
+  _getBorderZIndex = () => {
+    return defaultZIndex + 2
+  }
+  /**
+   * Set year which will be displayed on title according to parameter
+   * @param {number} currentYear
+   */
+  _getYear = (currentYearIndex) => {
+    this.setState({
+      currentYear: yearList[currentYearIndex]
+    })
+  }
+  /**
+   * The function is used by list.js for computing the height of children in timeline after they are mounted
+   * @param {Array} contentHeightArray
+   */
+  _getYearContentHeight = (contentHeightArray) => {
+    this.setState({ yearContentHeight: contentHeightArray })
+  }
+  componentDidMount() {
+    const timelineScrollingHeight = ReactDOM.findDOMNode(this.scrollingContent).getBoundingClientRect().height
+    this.setState({ timelineScrollingHeight: timelineScrollingHeight })
+  }
+  render() {
+    return (
+      <Container>
+        <SectionWrapper>
+          <Title><span>大事紀</span></Title>
+          <Circle color={`${colors.black}`}>
+            <Circle color={`${colors.gray.gray96}`}/>
+            <Rect color={`${colors.secondaryColor}`}/>
+          </Circle>
+          <Content>
+            <AccordionTimeline>
+              <List
+                unfoldArray={this.state.unfoldArray}
+                sortedData={sortedData}
+                sortedDataGroupByYear={sortedDataGroupByYear}
+                foldAndUnfold={this._foldAndUnfold}
+                getYearContentHeight={this._getYearContentHeight}  
+              />
+            </AccordionTimeline>
+            <YearRange>
+              <p><span>{yearList[yearList.length - 1]}</span></p>
+              <p>
+                <span>{yearList[0]}</span>
+                <img src={arrow} />
+              </p>
+            </YearRange>
+            <RunningTimeline>
+              <Timeline
+                ref={scrollingContent => this.scrollingContent = scrollingContent}
+                childrenHeight={this.state.timelineScrollingHeight * timelineScrollingPortion} 
+                autoScrolling={this.state.timelineScrolling}
+                startAutoScroll={this._startTimelineAutoScrolling}
+                stopAutoScroll={this._stopTimelineAutoScrolling}
+                yearContentHeight={this.state.yearContentHeight}
+                getYear={this._getYear}>
+                <List
+                  unfoldArray={this.state.unfoldArray}
+                  sortedData={sortedData}
+                  sortedDataGroupByYear={sortedDataGroupByYear}
+                  foldAndUnfold={this._foldAndUnfold}
+                  getYearContentHeight={this._getYearContentHeight}  
+                />
+              </Timeline>
+            </RunningTimeline>
+            <YearTag>
+              <p>{this.state.currentYear}</p>
+            </YearTag>
+          </Content>
+        </SectionWrapper>
+        <BorderBottom 
+          fixed={this.state.isBorderBottomfixed}
+          zIndex={this._getBorderZIndex()}
```

Comment:
why do we need `BorderBottom` fixed ?

Reply:
Since we need to see the border-bottom line of the container before we scrolling to the bottom of page. So I make the border-bottom line to be `fixed` before entering section5, and changes the position to `absolute` inside the container of section5.